app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
+        nonlocal cache_storage
+        if args in cache_storage:
+            print("Getting from cache")
+            return cache_storage[args]
+        else:
+            func_result = func(*args, **kwargs)
+            cache_storage.update({args: func_result})
```

Comment:
Similar to the previous comment, ensure that `args` is converted to a hashable type before using it as a key in `cache_storage`.